### PR TITLE
Update students.js

### PR DESCRIPTION
**Pull request by DanielSantoscoy**
Added my profile.

### CODE REVIEW

**Comment by Thomas-JC** [comment]
looks good

**Review by dsl2022 — COMMENTED** [review]
why is this file deleted?

**Review by dsl2022 — COMMENTED** [review]
there is a merge conflict, you need to resolve it

**Comment by DanielSantoscoy** [comment]
> why is this file deleted?

Sorry David I believe I merged from wrong branch which was the problem. Let me try again from the other branch
